openff/toolkit/topology/topology.py

```diff
@@ -610,13 +644,46 @@ def is_periodic(self, is_periodic):
             )
 
     @property
-    def constrained_atom_pairs(self):
+    def charge_model(self):
+        """
+        Get the partial charge model applied to all molecules in the topology.
+
+        Returns
+        -------
+        charge_model : str
+            Charge model used for all molecules in the Topology.
+
+        """
+        return self._charge_model
```

Comment:
Sorry, I may have screwed something up in the merge conflict resolution, because I can't reason around this part of the diff. Could you remake this branch/PR? This is totally my fault for taking so long that this fell into conflict.

Reply:
This is a red herring, it was removed in #1213 and founds its way back via some merge

Reply:
Did some fiddling with my local branch, things are a bit cleaner now.

Reply:
Thanks! This is much more readable now.